CircleandCross2/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+
+namespace CircleandCross2
+{
+	class Program
+	{
+
+		static void Main(string[] args)
+		{
+			try
+			{
+
+
+				Console.WriteLine("Tic Tac Toe");
+
+				string[,] array = new string[3, 3] { { "[ ]", "[ ]", "[ ]" }, { "[ ]", "[ ]", "[ ]" }, { "[ ]", "[ ]", "[ ]" } };
+
+				for (int i = 0; i < array.GetLength(0); i++)
+				{
+					for (int j = 0; j < array.GetLength(1); j++)
+					{
+						string arr = array[i, j];
+						Console.Write(arr);
+					}
+					Console.WriteLine();
+				}
```

Comment:
Duplicated code for displaying game board. Please introduce a function for displaying the board and use it everywhere.

Reply:
I do this

Reply:
Nice. Change the name of the function to contain a verb. Something like ShowGameboard or Display-